Dynatrace/dynatrace-operator
# Add WEBHOOK_PORT environment variable

## Description

Allows users to configure port webhook listens on via environment variable WEBHOOK_PORT. Default of 8443 is kept if environment variable is unset or invalid.

Resolves https://github.com/Dynatrace/dynatrace-operator/issues/2623

This change will allow users in EKS clusters using overlays to configure the port the webhook listens on to avoid collisions with other webhooks.

## How can this be tested?

Running the application with `WEBHOOK_PORT` set to a valid integer should cause the webhook to listen on the provided port.

Running the application without `WEBHOOK_PORT` set, or set to an invalid value, should cause the webhook to continue listening on `8443`.


## Checklist
